unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -285,3 +286,9 @@ function cert_setpermisions() {
     eval "chmod 500 /tmp/wazuh-certificates ${debug}"
     eval "chmod 400 /tmp/wazuh-certificates/* ${debug}"
 }
+
+function cert_convertCRLFtoLF() {
+    eval "( config_path="$(dirname "$(readlink -f "$1")")" )"
```

Comment:
Temporary files must be created in `/tmp/wazuh-install-files`

Reply:
Done in [commit](https://github.com/wazuh/wazuh-packages/pull/1426/commits/99c8a1f7d5d7dae80616871b7cf2800bcb902a0c)
Done in [commit](https://github.com/wazuh/wazuh-packages/pull/1426/commits/f375d8e444845bc74670b188b4d9afa7d969ff67)